runtime/assembly/api/tests/storage.ts

```diff
@@ -25,8 +25,47 @@ export function test_storage_set(data: i32, len: i32): i64 {
  * @param data - i32 pointer to the start of the arguments passed
```

Comment:
this should be "Test the get method of Storage module" and not "Test set method of storage"